packages/zend-mail/library/Zend/Mail/Storage/Mbox.php

```diff
@@ -335,7 +335,9 @@ protected function _openMboxFile($filename)
      */
     public function close()
     {
-        @fclose($this->_fh);
+        if ($this->_fh && 'Unknown' !== get_resource_type($this->_fh)) {
+            @fclose($this->_fh);
```

Comment:
@Megatherium, @falkenhawk should the @-silencer still be applied here?

Reply:
I'd probably keep `@` just to be extra safe...
Btw, should other `@fclose` in this class be similarly wrapped, or maybe `$this->close()` should be used in other places? 🤔

Reply:
@Megatherium: Why was this specific `fclose()` even addressed?

Perhaps the `close()` method lacks `$this->_fh = null;` to avoid errors on double close?

Reply:
@falkenhawk The `close()` method also resets `$this->_positions`, not sure what's the side effect. but seems it's called on open sequence so probably safe to do so...

... and the other call uses private handle

Reply:
I think calling close instead of close resulted these failures:
- https://github.com/zf1s/zf1/runs/2114162778

will revert the commit and see :)